test/e2e_node/device_plugin_test.go

```diff
@@ -282,7 +280,7 @@ func testDevicePlugin(f *framework.Framework, pluginSockDir string) {
 			gomega.Expect(v1ResourcesForOurPod.Containers[0].Devices[0].DeviceIds).To(gomega.HaveLen(1))
 		})
 
-		f.It("can make a CDI device accessible in a container", feature.DevicePluginCDIDevices, func(ctx context.Context) {
+		f.It("can make a CDI device accessible in a container", func(ctx context.Context) {
```

Comment:
it may need to be marked as a NodeConformance since it diesn't require any special environments